docs/index.md

```diff
@@ -62,7 +62,68 @@ We expect you will prioritize some of these features over others as you make Cat
+Native integrations developed for Catalyst are made available as Git patches that the Catalyst CLI can automatically apply to your newly created storefront.
+
+In order to create your Catalyst storefront with an out-of-the-box integration, run:
+
+```bash
+pnpm create @bigcommerce/catalyst@latest --integration=integration-name
+```
+
+Replace `integration-name` with the folder name of a folder inside the `integrations` directory of the Catalyst monorepo: https://github.com/bigcommerce/catalyst/tree/main/integrations
+
+For example, you can create a Catalyst storefront with the Makeswift integration by running:
+
+```bash
+pnpm create @bigcommerce/catalyst@latest --integration=makeswift
+```
+
+### Developing a native integration for Catalyst
+
+We support and encourage any third-party contributions for Catalyst, especially those introducing new integrations, or improving existing integrations. 
+
+At a high level, developing an integration for Catalyst typically involves the steps listed below:
+
+#### 1. Fork the Catalyst Monorepo
+
+Begin by forking the [`bigcommerce/catalyst` repository](https://github.com/bigcommerce/catalyst), clone the fork locally, and [follow the steps to get started with local monorepo development](https://www.catalyst.dev/docs/monorepo).
+
+#### 2. Create a Branch to Build Your Integration
+
+Next, create a branch based off the latest `@bigcommerce/catalyst-core` release tag and name it following the convention `integrations/your-integration-name`. This branch is where you'll be building your integration into Catalyst, and the naming convention helps us organize all integration branches inside the Catalyst monorepo.
+
+```bash
+git checkout -b integrations/your-integration-name @bigcommerce/catalyst-core@X.X.X
+```
+
+All integrations should be based off the latest `@bigcommerce/catalyst-core` release tag. This is because new Catalyst projects are created based off that tag, and if your integration is compatible with the latest tag, then it will be compatible with all Catalyst projects created from that tag.
+
+> [!IMPORTANT]
+>
+> #### Things to consider when building integrations:
+>
+> - In order to ensure your integration applies cleanly to new Catalyst projects, your integration should be 100% contained within the `core` folder of the monorepo. With the exception of installing packages inside of `core` (which in turn modifies the root `pnpm-lock.yaml` file), none of your integration code should live outside of the `core` folder.
+> - If your integration requires environment variables to work, be sure to add those environment variables to `core/.env.example`. This allows the `integrate` CLI to track which environment variables are required for the integration to work.
```

Comment:
Will Github flavored markdown render properly on `catalyst.dev`?

Reply:
It should